core/src/main/java/hudson/model/DownloadService.java

```diff
@@ -407,13 +409,17 @@ public FormValidation updateNow() throws IOException {
                 if (signatureCheck) {
                     FormValidation e = new JSONSignatureValidator("downloadable '"+id+"'").verifySignature(o);
                     if (e.kind!= Kind.OK) {
+                        LOGGER.log(Level.WARNING, "signature check failed for " + site, e );
                         continue;
                     }
                 }
                 jsonList.add(o);
             }
-            if (jsonList.size() == 0) {
-                return FormValidation.warning("None of the Update Sites passed the signature check");
+            if (jsonList.size() == 0 && toolInstallerMetadataExists) {
+                return FormValidation.warning("None of the tool installer metadata passed the signature check");
+            } else if (!toolInstallerMetadataExists) {
+                LOGGER.log(Level.WARNING, "No tool installer metadata found for " + id);
+                return FormValidation.ok();
```

Comment:
mmm not really, the experimental update centers for example does not have any and same goes for current and others..